### PR TITLE
Improvements for search engine data check

### PR DESCRIPTION
**Pull request by simahawk**
Depends on https://github.com/OCA/search-engine/pull/53 (not included)

### CODE REVIEW

File: shopinvader_search_engine/models/ir_exports_line.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IrExportsLine(models.Model):
+    _inherit = "ir.exports.line"
+
+    # Allow to turn on/off index data for products and categories
+    active = fields.Boolean(string="Active", default=True)
```
Comment (i-vyshnevska) [inline shopinvader_search_engine/models/ir_exports_line.py]: "ir.exports.line" don't have views? how they will enable/disable it?

Comment (simahawk) [reply shopinvader_search_engine/models/ir_exports_line.py]: nope, they don't. ATM this is just for technical usage (disable things via custom modules).

**Review by i-vyshnevska — APPROVED** [review]

File: shopinvader_search_engine/models/ir_exports_line.py (hunk 1 above)
Comment (simahawk) [reply shopinvader_search_engine/models/ir_exports_line.py]: FTR moved here https://github.com/OCA/server-tools/pull/1904

**Comment by simahawk** [comment]
to be rebased as #696 get merged

**Pull request merged**